src/commcare_cloud/ansible/roles/formplayer/tasks/main.yml

```diff
@@ -54,24 +54,44 @@
     - formplayer_deploy
   with_items:
     - "{{ formplayer_build_dir }}"
-    - "{{ formplayer_release_dir }}/libs"
+    - "{{ formplayer_release_dir }}"
     - "{{ formplayer_data_dir }}"
 
 - name: Download formplayer release
   get_url:
     url: "{{ formplayer_build_url }}"
-    dest: "{{ formplayer_release_dir }}/libs/formplayer.jar"
+    dest: "{{ formplayer_release_dir }}/formplayer.jar"
     mode: 0744
     owner: "{{ cchq_user }}"
     group: "{{ cchq_user }}"
   tags:
     - formplayer_deploy
 
+- name: Check if the "current" directory exists
+  stat:
+    path: "{{ formplayer_build_dir }}/current"
+  register: current_stat
```

Comment:
should this have `changed_when: false`?

Reply:
Oh, does `stat` fail if the path isn't there? I assumed it just set `.stat.exists` to false.

Reply:
No, but ansible will detect it as a change (and note it as such in the final report) when in fact it was just a check to set a local variable.

Reply:
Oh! `changed_when`, I was obviously thinking `failed_when`.

Still though I'm not seeing it report this as a change

```
TASK [formplayer : Check if the "current" directory exists] ************************************************************************************************************************************************
ok: [10.201.10.234]
```

That's what I would expect (`stat` just pulls info about the path, so I wouldn't expect it to report a change), but it sounded like you were reporting that `stat` sometimes has weird behavior?